CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Adding `mobile-ui-testing-squad` as codeowners for UI tests related folders
+
```

Comment:
My two cents: I think we can do without this comment. The syntax is straightforward enough.

Reply:
Good point, I removed the comment in 09979cdace22591c80c534579fa3f93b1ae143d9. Thank you for reviewing!